TTTSwiftTests/TTTAcceptanceTests.swift

```diff
@@ -5,9 +5,9 @@ import TTTSwift
 class TTTAcceptanceTests: QuickSpec {
     override func spec() {
         
-        describe("a full game of tic-tac-toe between two first available space computers plays through and exits properly") {
+        describe("acceptance tests") {
```

Comment:
This describe seems redundant. It's probably going to be obvious what sort of test this is based on some combination of Target/Group/file the test is in.

Try something like:
```
describe "Playing a full game between two simple computers"
    it "ends with the first computer winning"
```